server/command_restart.go

```diff
@@ -6,23 +6,42 @@ import (
 	cloud "github.com/mattermost/mattermost-cloud/model"
 	"github.com/mattermost/mattermost-server/v6/model"
 	"github.com/pkg/errors"
+	flag "github.com/spf13/pflag"
 )
 
+func getRestartFlagSet() *flag.FlagSet {
+	restartFlagSet := flag.NewFlagSet("restart", flag.ContinueOnError)
+	restartFlagSet.Bool("shared-installation", false, "Set this to true when attempting to restart a shared installation")
+
+	return restartFlagSet
+}
+
 func (p *Plugin) runRestartCommand(args []string, extra *model.CommandArgs) (*model.CommandResponse, bool, error) {
 	if len(args) == 0 || len(args[0]) == 0 {
 		return nil, true, errors.New("must provide an installation name")
 	}
 
+	restartFlagSet := getRestartFlagSet()
+	err := restartFlagSet.Parse(args)
+	if err != nil {
+		return nil, true, err
+	}
+
+	includeShared, err := restartFlagSet.GetBool("shared-installation")
```

Comment:
Do we need this? The shared installations already have `AllowSharedUpdates`, so it seems arbitrary to require that this flag is passed. We already validate that you are authorized to make the update, so can we just have it be automatic regardless of installation shared vs owned?

Reply:
I'm in favour of doing that, but I'm assuming the flag was added to address a specific concern re: `update`. Maybe the names are globally unique? @gabrieljackson, do you have context?

Reply:
I am good with this as is. If we want to remove it that's also fine -- just re-request a review when done.